staff/sean-daryanani/cv.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sean Curriculum</title>
+</head>
+
+<body>
+    <header>
+        <h1>Sean Daryanani | Aspiring Web Developer</h1>
+        <address>
+            <p><strong>Calle</strong>: XXXXXXXXXXX</p>
```

Comment:
`strong` element here it seems to be used to mark titles in bold, but that's NOT the correct purpose of `strong` element, please RTFM https://www.w3schools.com/tags/tag_strong.asp